### ISSUE TITLE
Enable source maps for production if possible

### ISSUE THREAD

**Issue by maxceem** [appirio-tech/connect-app#2534]
Follows this one https://github.com/appirio-tech/connect-app/issues/2435

Comment from @vikasrohit:
> do you think we could have any potential performance issue if we enable source map on production?

**Comment by maxceem** [comment]
@vikasrohit there is one moment about the source maps. Is it ok for them to be exposed to everyone? I mean if we add source maps on production everybody could read the source code of our app which potentially could lead to security issues. On the other hand the repository for Connect App is open for everyone so maybe it's not an issue for us. 

What do you think?

**Comment by maxceem** [comment]
@vikasrohit maybe we can give another try to make this enhancement. I think it would be nice to have it to make it easier to debug issues happened on prod https://github.com/appirio-tech/connect-app/issues/2791, https://github.com/appirio-tech/connect-app/issues/2900, https://github.com/appirio-tech/connect-app/issues/2899. Though still not critical as we can also get the error places using the obfuscated code, just takes more efforts.

**Comment by vikasrohit** [comment]
Agreed. However, in most of the issues reported by Sentry, we are getting exact place where the issue was raised and even with the partially obfuscated code, I was able to locate the exact code in our repo as well.

**Comment by RishiRajSahu** [comment]
@maxceem do we have any open question on this one, I guess we are already covering sentry issues in this CF.

**Comment by maxceem** [comment]
The main question is if we should enable source maps on production or no.

In most cases, we can find the place of error. Though it feels that it would be more comfortable if we have it enabled.
